react/src/components/dropdown/SprkDropdown.js

```diff
@@ -282,8 +304,14 @@ SprkDropdown.propTypes = {
         href: PropTypes.string,
         /** The text inside the choice item. */
         text: PropTypes.string,
+        /**
+         * If set to true, current option will be set as default selected option
+         *  */
+        default: PropTypes.bool,
```

Comment:
Hey! Thanks for the PR. Could you change this to isDefault?

Reply:
have updated the pull request with asked changes